static/templates/invite_user.hbs

```diff
@@ -30,6 +30,19 @@
                     </div>
                 </div>
                 <div class="input-group">
+                    <label for="expires_in">{{t "Invitation expires after" }}</label>
```

Comment:
Should this new input gets its own `input-group` wrapping div?

Reply:
Yes of course. I fixed it.